tests/PHPUnit/System/expected/test_apiGetReportMetadata__API.getReportMetadata_day.xml

```diff
@@ -1808,10 +1808,11 @@
 	</row>
 	<row>
 		<category>Visitor Devices</category>
-		<name>Device brand</name>
```

Comment:
Again Visitor Browser and Device model both have the same report order id. Therefore it switches sometimes! This is an issue that I do not want to fix here.